internal/clientgen/clientgentypes/clientgentypes.go

```diff
@@ -63,3 +64,56 @@ func NewServiceSet(md *meta.Data, include, exclude []string) ServiceSet {
 func AllServices(md *meta.Data) ServiceSet {
 	return NewServiceSet(md, []string{"*"}, nil)
 }
+
+type TagSet struct {
+	tagMap       map[string]bool
+	includedTags []string
```

Comment:
I'd much rather model this as two separate maps:

```go
included map[string]bool
excluded map[string]bool
```

rather than having a single map represent both and then a separate list for the included tags. Seems much easier to reason about to me

Reply:
No problem, done